web-ng/src/app/shared/styles/_variables.scss

```diff
@@ -25,6 +25,9 @@
 $primary: mat-palette($mat-green, 700, 600, 800);
 $accent: mat-palette($mat-amber, 700, 600, 800);
 $warn: mat-palette($mat-pink, 600, 500, 700);
+$white: mat-palette($mat-grey, A100);
```

Comment:
Please define custom color values below in `:root` instead since these aren't actually used by the theming component.

Reply:
removed